engine/api/src/main/java/io/deephaven/engine/table/iterators/ColumnIterator.java

```diff
@@ -12,13 +12,9 @@
  * @apiNote ColumnIterators must be explicitly {@link #close() closed} or used until exhausted in order to avoid
  *          resource leaks.
  */
-public interface ColumnIterator<DATA_TYPE> extends CloseableIterator<DATA_TYPE>, SafeCloseable {
+public interface ColumnIterator<DATA_TYPE> extends ValueIterator<DATA_TYPE>, SafeCloseable {
 
     @Override
     default void close() {}
```

Comment:
Why are we keeping this default?

Reply:
Discussed offline, this is because of the diamond inheritance.